modules/sdk/src/main/scala/org/tessellation/sdk/domain/collateral/LatestBalances.scala

```diff
@@ -0,0 +1,8 @@
+package org.tessellation.sdk.domain.collateral
+
+import org.tessellation.schema.address.Address
+import org.tessellation.schema.balance.Balance
+
+trait LatestBalances[F[_]] {
+  def getLatestBalances: F[Option[Map[Address, Balance]]]
```

Comment:
IMO it's pointless to expose that. It can be an internal function of Collateral service that would depend on global snapshot storage

Reply:
@kriscieslak please take a look here. our discussion related to LatestBalances trait could be irrelevant if above is true

Reply:
As we discussed `LatestBalances` interface is left to be implemented in `core` and `dag-l1`.